packages/server/modules/workspaces/graph/resolvers/projects.ts

```diff
@@ -0,0 +1,52 @@
+import { db } from '@/db/knex'
+import { Resolvers } from '@/modules/core/graph/generated/graphql'
+import { getFeatureFlags } from '@/modules/shared/helpers/envHelper'
+import { getPaginatedItemsFactory } from '@/modules/shared/services/paginatedItems'
+import { WorkspaceTeamMember } from '@/modules/workspaces/domain/types'
+import {
+  countInvitableCollaboratorsByProjectIdFactory,
+  getInvitableCollaboratorsByProjectIdFactory
+} from '@/modules/workspaces/repositories/users'
+
+const { FF_WORKSPACES_MODULE_ENABLED } = getFeatureFlags()
+
+export default FF_WORKSPACES_MODULE_ENABLED
+  ? ({
+      Project: {
+        invitableCollaborators: async (parent, args) => {
+          // TODO: add authz policy
+          if (!parent.workspaceId) {
+            return {
+              totalCount: 0,
+              items: [],
+              cursor: null
+            }
+          }
+
+          return await getPaginatedItemsFactory<
```

Comment:
if limit===0 only get count, dont invoke item retrieval query at all (same convention we have in other paginated resolvers)